### ISSUE TITLE
Broken on 52.0a1 (2016-10-18)

### ISSUE THREAD

**Issue by ChrisW-B** [denschub/firefox-tabgroups#53]
When trying to install on the latest nightly, tabgroups does not start up, instead leaving

```
1476837801460   addons.xpi  WARN    Error loading bootstrap.js for tabgroups@schub.io: Module `jar:file:///Users/chrisbarry/Library/Application%20Support/Firefox/Profiles/qrpb9k0m.default-1476835073930/extensions/tabgroups@schub.io.xpi!/lib/migrationoverride` is not found at resource://gre/modules/commonjs/jar:file:///Users/chrisbarry/Library/Application%20Support/Firefox/Profiles/qrpb9k0m.default-1476835073930/extensions/tabgroups@schub.io.xpi!/lib/migrationoverride.js (resource://gre/modules/commonjs/toolkit/require.js:61) JS Stack trace: make/require@require.js:61:12 < @jar:file:///Users/chrisbarry/Library/Application%20Support/Firefox/Profiles/qrpb9k0m.default-1476835073930/extensions/tabgroups@bootstrap.js:23:5 < @XPIProvider.jsm:4864:1 < this.XPIProvider.loadBootstrapScope@XPIProvider.jsm:4864:7 < this.XPIProvider.callBootstrapMethod@XPIProvider.jsm:4943:9 < AddonInstall.prototype.startInstall/<@XPIProvider.jsm:6509:11
1476837801462   addons.xpi  WARN    Add-on tabgroups@schub.io is missing bootstrap method install
1476837801604   addons.xpi  WARN    Add-on tabgroups@schub.io is missing bootstrap method startup
```

in the console.
It works fine if you run it with `env JPM_PARAMS="-b /Applications/FirefoxNightly.app/Contents/MacOS/firefox" ./node_modules/.bin/jake run`, but if you try to install a built xpi onto a clean profile, it does not start up, giving the above error

**Comment by jon-paysbuy** [comment]
Looks like Mozilla may have stripped out some Add-On SDK stuff that this was relying on 😞

**Comment by skyschub** [comment]
No worries. Fixed by 4239812f0c4b3b71f783cd84f324e6e1e07f8980, the new version is already on AMO to review.
